herbert_core/DLL/.gitignore

```diff
@@ -0,0 +1 @@
+mpiexec*
```

Comment:
what is this ignoring?

Reply:
MPI runtimes are copied here on build, so they can be packaged. This ignores `mpiexec.exe` on Windows and `mpiexec` on Linux